### PR TITLE
chore: rename 'Learn more' button text

### PR DESCRIPTION
**Pull request by ojafero**
# Description

Rename "Learn more" to a descriptive text for SEO.

# Checklist:

- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by jeremydthomas** [comment]
@ojafero could you throw up a picture of your work? Would help reviewers save some time to look at what you've done without loading up your code themselves, especially for a few lines of code.

File: src/components/Home.js [hunk 1]
```diff
@@ -138,7 +138,7 @@ export default function Home(props) {
                   },
                 }}
               >
-                Learn more
+                Read about Treetracker
```
Comment (jeremydthomas) [inline src/components/Home.js:141]: I think "Read More About Treetracker" or "Read More About The Treetracker System" would be a bit better. Its also better to capitalize the first letter of each word when it comes to seo. What do you think of those phrases @dadiorchen? This is a great step in the right direction though!

**Review by jeremydthomas — CHANGES_REQUESTED** [review]

**Comment by jeremydthomas** [comment]
Oh, I just noticed how old this PR is 😆

**Comment by dadiorchen** [comment]
Because this changed our UX, I will close this PR

**Pull request closed without merging**